x-pack/plugins/cases/server/client/configure/client.ts

```diff
@@ -258,15 +257,8 @@ async function update(
 
     const { version, ...queryWithoutVersion } = request;
 
-    /**
-     * Excess function does not supports union or intersection types.
-     * For that reason we need to check manually for excess properties
-     * in the partial attributes.
-     *
-     * The owner attribute should not be allowed.
-     */
     pipe(
-      excess(CasesConfigurePatchRt.types[0]).decode(queryWithoutVersion),
+      CasesConfigurePatchRt.types[0].decode(queryWithoutVersion),
```

Comment:
Let's actually remove this whole pipe and decode call. The call above should do the validation for us.